src/toast/instrument_sim.py

```diff
@@ -740,7 +744,8 @@ def fake_rhombihex_focalplane(
     xaxis, yaxis, zaxis = np.eye(3)
 
     # Gap between rhombi as a fraction of the total width
-    gap = 0.1 * width
+    if gap is None:
+        gap = 0.5 * width / int(np.sqrt(n_pix_rhombus) - 1)
```

Comment:
Why `int()`? Seems to me that the gap is already quantized by the number of pixels.

Reply:
Old C-programmer habits die hard :-)  I can remove it, especially since the gap is a float.